spec/features/journeys/happy_paths/when_get_an_identity_returns_no_trn_spec.rb

```diff
@@ -1,15 +1,15 @@
 require "rails_helper"
 
-RSpec.feature "Happy journeys", type: :feature do
+RSpec.feature "Happy journeys", :with_default_schedules, type: :feature do
   include Helpers::JourneyAssertionHelper
   include Helpers::JourneyStepHelper
   include ApplicationHelper
 
   include_context "retrieve latest application data"
-  include_context "Stub previously funding check for all courses" do
-    # In this situation we fallback to a non-pilot set of checks
-    let(:api_call_trn) { manually_entered_trn }
-  end
+  # include_context "Stub previously funding check for all courses" do
+  #   # In this situation we fallback to a non-pilot set of checks
+  #   let(:trn) { manually_entered_trn }
```

Comment:
Can we just delete these setup if its not used?

Reply:
I have actually removed that context on my pr:
https://github.com/DFE-Digital/npq-registration/pull/2042